conversations/v3/webhook-events-v2.yaml

```
@@ -755,6 +790,75 @@ components:
         whatsapp:
           senderName: open api sample
           text: I liked the event
+    MoMessageOrder:
+      type: object
+      description: |
+        Orders are triggered when a user places an order from a shopping cart.
+      required:
+        - messageId
+        - event
+        - status
+        - timestamp
+        - order
+      allOf:
+        - $ref: '#/components/schemas/APIEvent'
+      properties:
+        event:
+          type: string
+          description: Determines which event type is sent. Always _MoMessage::Order_
+          enum:
+            - MoMessage::Order
+        channel:
+          type: string
+          description: Channel that triggered the event.
+          example: whatsapp
+          enum:
+            - whatsapp
+        timestamp:
+          type: string
+          format: 'date-time'
+          description: Point in time when the message was received
+          example:
+            2019-06-26T11:41:00
+        messageId:
+          type: string
+          description: Message ID reference
+          example: 77185196-664a-43ec-b14a-fe97036c697f
+        from:
+          type: string
+          description: The sender of the message
+          example: "49123512314"
+        to:
+          type: string
+          description: Receiving account of the message. The format is channel specific.
```

```suggestion
          description: Receiving account of the message. The format is channel specific.
```
Could we use` Receiver. The format is channel specific` here?

---

yes, will apply :)